WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -168,6 +168,12 @@ extension UIImage {
             .imageFlippedForRightToLeftLayoutDirection()
     }
 
+    /// Empty Coupons Icon
+    ///
+    static var emptyCouponManagementListImage: UIImage {
+        return UIImage(named: "woo-error")!
+    }
+
```

Comment:
Thanks for the heads up. I have an image now but to avoid this PR getting bigger I'll switch to errorImage for the time being and update the empty state in a future PR, with tests for the new image.